app/react/src/client/preview/element_check.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// return true if the element is renderable with react fiber
+export const isValidFiberElement = element =>
+  typeof element === 'string' || typeof element === 'number' || React.isValidElement(element);
+
+// input: [1, 2, [3, 4, [5, 6, [7]]]]
+// output: [ 1, 2, 3, 4, 5, 6, 7 ]
+export const flattenList = list =>
```

Comment:
👌  definitely! I was hesitant to import `lodash` as I saw very few use of it in the project! 👍 will replace this function by `lodash.flattenDeep`!